tsconfig.json

```diff
@@ -22,7 +22,6 @@
     "resolveJsonModule": true,
     "composite": true,
     "lib": ["DOM", "ES2020"],
-    "isolatedDeclarations": true,
-    "noUncheckedIndexedAccess": true,
+    "noUncheckedIndexedAccess": true
```

Comment:
Removing `isolatedDeclarations` addressed failing type errors when importing Auto for the new auto plugin.